api/app/src/main/kotlin/packit/model/Packet.kt

```diff
@@ -1,16 +1,20 @@
 package packit.model
 
-import jakarta.persistence.*
+import jakarta.persistence.Convert
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
 import packit.helpers.JsonMapConverter
 
 @Entity
 data class Packet(
-        @Id
-        val id: String,
-        val name: String,
-        val displayName: String,
-        @Convert(converter = JsonMapConverter::class)
-        val parameters: Map<String, Any>,
-        val published: Boolean,
-        val time: Long
+    @Id
+    val id: String,
+    val name: String,
+    val displayName: String,
+    @Convert(converter = JsonMapConverter::class)
+    val parameters: Map<String, Any>,
+    val published: Boolean,
+    val importTime: Long,
+    val startTime: Double,
+    val endTime: Double
```

Comment:
Sorry, late to this party obviously, but is there a reason why we're keeping importTime in a different format than start and end time? Is it because they have a millisecond part? Might be good to be consistent with those, or comment on the distinction.